azurerm/internal/services/postgres/postgresql_flexible_server_configuration.go

```diff
@@ -0,0 +1,172 @@
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Importer: pluginsdk.ImporterValidatingResourceId(func(id string) error {
+			_, err := parse.FlexibleServerConfigurationID(id)
+			return err
+		}),
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"server_name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.FlexibleServerName,
+			},
+
+			"value": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceFlexibleServerConfigurationUpdate(d *pluginsdk.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Postgres.FlexibleServersConfigurationsClient
+	ctx, cancel := timeouts.ForUpdate(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	log.Printf("[INFO] preparing arguments for Azure Postgresql Flexible Server configuration creation.")
+
+	name := d.Get("name").(string)
+	resGroup := d.Get("resource_group_name").(string)
+	serverName := d.Get("server_name").(string)
+
+	props := postgresqlflexibleservers.Configuration{
+		ConfigurationProperties: &postgresqlflexibleservers.ConfigurationProperties{
+			Value:  utils.String(d.Get("value").(string)),
+			Source: utils.String("user-override"),
+		},
+	}
+
+	future, err := client.Update(ctx, resGroup, serverName, name, props)
+	if err != nil {
+		return err
+	}
+
+	if err = future.WaitForCompletionRef(ctx, client.Client); err != nil {
+		return err
+	}
+
+	resp, err := client.Get(ctx, resGroup, serverName, name)
+	if err != nil {
+		return err
+	}
+	if resp.ID == nil {
+		return fmt.Errorf("Cannot read Azure Postgresql Flexible Server configuration %s (resource group %s) ID", name, resGroup)
+	}
+
+	d.SetId(*resp.ID)
+
+	return resourceFlexibleServerConfigurationRead(d, meta)
+}
+
+func resourceFlexibleServerConfigurationRead(d *pluginsdk.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Postgres.FlexibleServersConfigurationsClient
+	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := parse.FlexibleServerConfigurationID(d.Id())
+	if err != nil {
+		return err
+	}
+
+	resp, err := client.Get(ctx, id.ResourceGroup, id.FlexibleServerName, id.ConfigurationName)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			log.Printf("[WARN] Azure Postgresql Flexible Server configuration '%s' was not found (resource group '%s')", id.ConfigurationName, id.ResourceGroup)
+			d.SetId("")
+			return nil
+		}
+
+		return fmt.Errorf("Error making Read request on Azure Postgresql Flexible Server configuration %s: %+v", id.ConfigurationName, err)
```

Comment:
We can omit the "Error" prefix of error messages since that's always printed.
```suggestion
		return fmt.Errorf("making Read request on Azure Postgresql Flexible Server configuration %s: %+v", id.ConfigurationName, err)
```

Reply:
fixed